src/entry/Entry.tsx

```diff
@@ -43,22 +43,31 @@ const Entry: React.FC<EntryProps> = ({ match }) => {
         content={(
           <Fragment>
             <div className="button-group">
-              <button type="button" className="button link-button">Blast</button>
-              <button type="button" className="button link-button">Align</button>
+              <button type="button" className="button link-button">
+                Blast
+              </button>
+              <button type="button" className="button link-button">
+                Align
+              </button>
               <button type="button" className="button link-button">
                 <DownloadIcon />
                 Download
               </button>
-              <button type="button" className="button link-button">Add</button>
+              <button type="button" className="button link-button">
+                Add
+              </button>
             </div>
-            <ProteinOverview
-              data={transformedData[EntrySection.NamesAndTaxonomy]}
-              proteinExistence={transformedData.proteinExistence}
-              primaryAccession={transformedData.primaryAccession}
-              annotationScore={transformedData.annotationScore}
-              uniProtId={transformedData.uniProtId}
-              entryType={transformedData.entryType}
-            />
+            <Card
```

Comment:
It uses an element for title rather than EntrySection and doesn't appear in the in-page nav - and might also en up not being in a card (TBD) so I'd rather not.